docs/product/insights/mobile/mobile-screens/index.mdx

```diff
@@ -0,0 +1,84 @@
+
+- [Android](/platforms/android/tracing/instrumentation/automatic-instrumentation/#app-start-instrumentation)
+- [Flutter](/platforms/flutter/tracing/instrumentation/automatic-instrumentation/#app-start-instrumentation)
+- [iOS](/platforms/apple/guides/ios/tracing/instrumentation/automatic-instrumentation/#app-start-tracking)
+- [React Native](/platforms/react-native/tracing/instrumentation/automatic-instrumentation/#app-start-instrumentation)
+
+In the example below, the detail view of a transaction displays the warm start measurement in the right sidebar.
+
+![The event detail of a transaction with a warm start measurement.](./img/app-start-transaction.png)
+
+While the SDKs differentiate between cold and warm starts, they don't track hot starts or resumes. To get more insight into your cold and warm start metrics, you can use the [App Starts](/product/insights/mobile/mobile-screens/app-starts/) feature.
+
+## Slow and Frozen Frames
+
+To track the responsiveness of the user interface, Sentry measures _slow frames_ and _frozen frames_. Typically, a phone or tablet renders 60 frames per second (fps). At 60 fps, every frame has 16 or 16.67 ms to render.
+
+- **Slow Frames**: Using 60 fps, slow frames are frames that take more than 16 ms (Android) or 16.67 ms (iOS) to render.
+- **Frozen Frames**: Frozen frames are frames that take longer than 700 ms to render.
+
+For Apple, the frame rate can be higher, especially as 120 fps displays are becoming more popular. For these apps, Sentry detects the frame rate and adjusts the slow frame calculation accordingly.
+
+On the mobile screens insights page, you can see a slow and frozen frame rate, indicating the percentage of frames that are slow or frozen during the lifetime of the transaction.
+
+In the example below, the detail view of the transaction displays the slow, frozen, and total frames in an iOS application:
+
+![The event detail of a transaction with slow and frozen frames measurements.](./img/slow-frozen-frames.png)
+
+You can track slow and frozen frames for:
+
+- [Android](/platforms/android/tracing/instrumentation/automatic-instrumentation/#slow-and-frozen-frames)
+- [Flutter](/platforms/flutter/tracing/instrumentation/automatic-instrumentation/#slow-and-frozen-frames)
+- [iOS](/platforms/apple/guides/ios/tracing/instrumentation/automatic-instrumentation/#slow-and-frozen-frames)
+- [React Native](/platforms/react-native/tracing/instrumentation/automatic-instrumentation/#slow-and-frozen-frames)
+
+## Frames Delay
+
+Frames Delay is the user-perceived delayed duration of rendered frames. For example, at a refresh rate of 60fps, if a single frame render takes 20ms, the frame delay is 4ms (20ms - 16ms = 4ms).
```

Comment:
I think it would be great to add another example with two or more frames to clearly highlight it's not only about a single frame.